app/views/admin/teams/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1 class="govuk-heading-xl"><%= t('team.heading') %></h1>
+
+<% if @teams.present? %>
+  <table class="govuk-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="col"><%=t('team.name')%></th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <% @teams.reverse.each do |team| %>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell" scope="row"><%= team.name %></td>
+        </tr>
```

Comment:
it would be awesome if we could list the components associated with the teams here... but we might leave it for the next PR

Reply:
Agreed